handbooks/learners-meetups.md

```diff
@@ -80,9 +89,15 @@ People, venue, drinks, a projector: the organizer's checklist!
  * Find attendees. We can help! If it doesn't already exist, we will set up an OTS Meetup account for your city (e.g. meetup.com/opentechschool-yourcity) and you'll be able to use it to communicate about the meetup and have people join the event. Furthermore, we'll promote the event page through our social media channels, mailing lists, user groups, and communities. Of course we also need _you_ to spread the word and get people to join! A couple of days before the event, write all the attendees a reminder e-mail asking them to update their RSVP status (if they are unable to join, by changing their RSVP to "not coming," there will be seats freed for people who might be on the waiting list).
 
 ## At the meetup
- * Be sure to get to the venue at least one hour in advance and with at least another co-organizer, in order to set up the space and the projector, as well as meet the speakers in advance.
+ * Be sure to get to the venue at least one hour in advance and with at least another co-organizer, in order to set up the space and the projector, put signs up outside, as well as meet the speakers in advance.
  * When the attendees start to arrive, someone from the organizational team should greet them and ask them to create and wear their nametags (stickers or simply tape to write on are both great for this).
- * Have a member of the OTS team moderate the event. When everybody is sitting, they should quickly introduce OTS, the meetup's structure and timetable, welcome the speakers on stage and lead the question time, encouraging everyone to feel at ease with sharing their questions. This person should also announce the beginning and end of the break, as well as moderate the Learn'n'Tell session.
+ * Have a member of the OTS team moderate the event. When everybody is sitting, they should do a quick introduciton covering:
+   * what is OTS?
+   * what is the Learners Meetup?
+   * explain that the event is meant to be especially welcoming for beginners
+   * the meetup's format and timetable
+   * point out the other organizers and coaches
+   * welcome the speakers on stage and lead the question time, encouraging everyone to feel at ease with sharing their questions.
```

Comment:
Much better. I wonder if we should link to one of the "What is OTS" slidedecks here.